cmd/telegraf/cmd_plugins.go

```diff
@@ -10,8 +10,10 @@ import (
 	"github.com/influxdata/telegraf/plugins/aggregators"
 	"github.com/influxdata/telegraf/plugins/inputs"
 	"github.com/influxdata/telegraf/plugins/outputs"
+        "github.com/influxdata/telegraf/plugins/parsers"
 	"github.com/influxdata/telegraf/plugins/processors"
```

Comment:
```suggestion
	"github.com/influxdata/telegraf/plugins/outputs"
    "github.com/influxdata/telegraf/plugins/parsers"
	"github.com/influxdata/telegraf/plugins/processors"
```

Reply:
Shouldn't have tried to do it from my phone hehe.

Reply:
Hmm did my proposal not work?

Reply:
No, doesn't look like it. Hopefully it works now.